tests/h/auth/util_test.py

```diff
@@ -2,106 +2,24 @@
 
 from __future__ import unicode_literals
 
-import base64
 from collections import namedtuple
-from h._compat import unichr
 
 import pytest
 import mock
-from hypothesis import strategies as st
-from hypothesis import given
 import sqlalchemy as sa
 
 from pyramid import security
 
 from h.auth import role
 from h.auth import util
 from h._compat import text_type
-from h.exceptions import ClientUnauthorized
 from h.models.auth_client import GrantType
 from h.models import AuthClient
-from h.schemas import ValidationError
 from h.services.user import UserService
 
 FakeUser = namedtuple('FakeUser', ['authority', 'admin', 'staff', 'groups'])
 FakeGroup = namedtuple('FakeGroup', ['pubid'])
 
-# The most recent standard covering the 'Basic' HTTP Authentication scheme is
```

Comment:
We were previously doing a whooooole lot of testing on the specifics of HTTP Basic Auth. As we are now no longer rolling our own auth handling here but instead using Pyramid's built-in Basic Auth Policy (as part of the AuthClientPolicy), I believe we can trust Pyramid's HTTP Basic Auth to work right. But I wanted to point this out.